e2e/framework/config.go

```diff
@@ -42,8 +43,9 @@ func (t *TestServerConfig) Premine(addr types.Address, amount *big.Int) {
 		t.PremineAccts = []*SrvAccount{}
 	}
 	t.PremineAccts = append(t.PremineAccts, &SrvAccount{
-		Addr:    addr,
-		Balance: amount,
+		Addr:          addr,
+		Balance:       amount,
+		StakedBalance: big.NewInt(0),
```

Comment:
@lazartravica @Vuksan @Kourin1996 
What do you guys think about adding the ability to set premined staked balance for accounts?

Reply:
That seems reasonable to me.